### PR TITLE
WhyFvwm layout

### PR DESCRIPTION
**Pull request by rasatpc**
New layout with correct html.

### CODE REVIEW

File: Wiki/NewToFvwm/WhyFvwm/index.md [hunk 1]
```diff
@@ -10,27 +10,35 @@ Fvwm is a very old project and at one time was a fairly
 popular window manager, and is now a tiny project still used
 by lots of core users. What is it about fvwm that users like?
 
-+ Fvwm is light weight and fast. Fvwm is built on top of X11,
+{% capture fvwmtxt %}
+<p><strong>Fvwm is light weight and fast.</strong> - Fvwm is built on top of X11,
```
Comment (ThomasAdam) [inline Wiki/NewToFvwm/WhyFvwm/index.md:14]: Remind me again why we need this HTML markup here?  Can't we do this just with Markdown?

```
**Fvwm is lightweight and fast**
```

Also, I would correct "light weight" to "lightweight".

File: Wiki/NewToFvwm/WhyFvwm/index.md [hunk 2]
```diff
@@ -10,27 +10,35 @@ Fvwm is a very old project and at one time was a fairly
...
   and doesn't use a lot of additional libraries or widget
   sets, as such it has a very small foot print on top of
-  the already running X server, making it quite sleek.
+  the already running X server, making it quite sleek.</p>
 
-+ Fvwm is very stable, and preforms its task as a window manager
-  very well. It is very rare fvwm crashes, and most instability
-  is the application itself.
+<p><strong>Fvwm is very stable and performs its task as a window manager
+  very well.</strong> - It is very rare fvwm crashes, and most instability
```
Comment (ThomasAdam) [inline Wiki/NewToFvwm/WhyFvwm/index.md:20]: Same thing here -- doesn't Markdown allow for this?

File: Wiki/NewToFvwm/WhyFvwm/index.md [hunk 3]
```diff
@@ -10,27 +10,35 @@ Fvwm is a very old project and at one time was a fairly
...
+  is the application itself.</p>
 
-+ Fvwm is extremely configurable, and you can customize almost
-  every possible aspect of your desktop. Though this does take
+<p><strong>Fvwm is extremely configurable.</strong> - You can customize almost
```
Comment (ThomasAdam) [inline Wiki/NewToFvwm/WhyFvwm/index.md:23]: No need for a hyphen after the sentence ends.

```
**Fvwm is extremely configurable.**  You can customize...
```

File: Wiki/NewToFvwm/WhyFvwm/index.md [hunk 4]
```diff
@@ -45,15 +53,19 @@ by lots of core users. What is it about fvwm that users like?
...
 As you learn how to tell fvwm how to handle your windows,
 you can tweak fvwm into your own dream window manager. Okay,
 there are limits as to what fvwm can do, but within its
 structure, there is a lot of power.
 
-# What are fvwm's weaknesses?
+<em><strong>I like fvwm’s approach</strong> - no defaults and everything is
+up to the user. I can hardly imagine myself using anything else.
+Even with my lame coding style, fvwm meets my needs perfectly.</em>
+
```
Comment (ThomasAdam) [inline Wiki/NewToFvwm/WhyFvwm/index.md:67]: Doesn't Markdown already allow for this?

```
**I like fvwm's approach** *this is in italics*
```

File: Wiki/NewToFvwm/WhyFvwm/index.md [hunk 5]
```diff
@@ -66,17 +78,17 @@ be used as a starting point, such as the [DefaultConfig](
...
-Fvwm's isn't built using any modern widget set or frame work, and
+Fvwm's isn't built using any modern widget set or framework, and
 though its modules can allow some building of interactive UI elements,
 they are a bit primitive and don't contain a lot of modern features.
-Though you can build tools using modern frame works that talk to
+Though you can build tools using modern frameworks that talk to
 fvwm.
 
-Fvwm isn't for everyone, but if a quick, fast, low memory use, and
+Fvwm isn't for everyone, but if a quick, fast, low-memory use and
```
Comment (ThomasAdam) [inline Wiki/NewToFvwm/WhyFvwm/index.md:91]: No need to make "low memory" compound here -- it's fine as-is.

**Review by ThomasAdam — CHANGES_REQUESTED** [review]
Hi @rasatpc 

Thanks for this.  Aside from the spelling mistakes you've noticed (some of which I don't believe need changing), I'm still a little confused about the HTML markup you're suggesting we add.

AFAICT, it seems to me that using pure markdown for this already does what we want it to do -- and in those cases, I've added suggestions for how I think that probably works.

File: Wiki/NewToFvwm/WhyFvwm/index.md (hunk 2 above)
Comment (somiaj) [reply Wiki/NewToFvwm/WhyFvwm/index.md:20]: We are inside a {% capture %} environment, and markdown isn't parsed for this. I wish I could use custom plugins on github, my sites all have a {% markdown %} environment I can force mark down rendering, but in this case we cannot do that. So unless there is a way to force markdown parsing, inside the FvwmWindows we need to use HTML.

Comment (ThomasAdam) [reply Wiki/NewToFvwm/WhyFvwm/index.md:20]: Riiight.  Then I guess the only thing to do here is to build it "locally" and push the result out to gh-pages.

Of course, "build it locally" could also mean in a mini container on Github itself which we use to publish the results -- that way, we can do whatever we like which would get rid of those restrictions.

**Comment by somiaj** [comment]
@rasatpc just some additional notes, part of `FvwmWindow` is the border, it really wasn't meant to be used as you did. I added a window around it. I also used a quote for the quote you added, nice touch, a single quote like that seems just fine.

I did realize that bold didn't stand out to well in FvwmWindow, so I did update the css to make bold use the same color as links do to stand out a bit more.

File: Wiki/NewToFvwm/WhyFvwm/index.md (hunk 2 above)
Comment (somiaj) [reply Wiki/NewToFvwm/WhyFvwm/index.md:20]: I have thought of the just uploading the html we build (this means we could also dig out my old syntax hilighter), but didn't think we could automate that. We should look into a setup like that, I would love to move the whole FvwmWindow from an include (which is why we have to use it in this strange way) to an actual plugin, that we can just have a start tag and end tag, and everything inbetween gets rendered using markdown.

**Comment by rasatpc** [comment]
> using pure markdown for this already does what we want it to do

We can not use markdown `**bold**` within FvwmWindow. `<HTML Content of Window>`
https://www.fvwm.org/Wiki/FvwmWeb/FvwmWindow/

Without HTML
https://rasatpc.net/screenshot/wiki/markdown.png

Any other suggestions?

**Comment by somiaj** [comment]
@rasatpc as per my previous remark, due to how I had to implement the FvwmWindow as in include (not a plugin), we have to capture raw text (which needs to be in html) and feed it to the inside of the window container. No rendering of markdown is done. Though go check out your updates applied with my tweaks, you can look at what I did in the last patch to get the window to show up better.

**Comment by rasatpc** [comment]
Thanks @somiaj, the page looks good.

**Pull request merged**